.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+---
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag'
+        required: true
+      artifacts-name:
+        description: 'Artifacts name to promote'
+        required: true
+
+jobs:
+  verify-release:
+    name: Verify if tag is valid
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      
+      - name: Fetch tags
+        run: git fetch --tags
+      
+      - name: 'Check if tag was already created'
+        shell: bash
+        run: >
+          git show-ref --tags v${{ github.event.inputs.tag }} --quiet \
+          && exit 1 || exit 0
```

Comment:
Why don't you create it here? ;)

```yaml
      - name: Create Release
        uses: softprops/action-gh-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          name: Release ${{ github.event.inputs.tag }}
          tag_name: ${{ github.event.inputs.tag }}
          generate_release_notes: true
          target_commitish: ${{ github.sha }}
```          

Haven't checked but if there's a release document, update it. If there's none. Create one.

Reply:
I am not sure I understand your comment: this first step is here to ensure we don't recreate an existing tag, before creating the release and promoting the artifacts, so it needs to be done before both. AFAIK, this is not supported by `softprops/action-gh-release@v1`, is it?

I will update the documentation!

Reply:
Here we go: https://github.com/scality/Zenko/pull/1717/commits/c060b83e3948d84166f9e0a68900dfde1b99d3ea#diff-87479ac9ea16cec5ceafce00b3b049d7d5920cb663afae42901f20e7c4e6705dR7

This is not yet tested, but I think that's the first thing I will do once we merge this PR, with test tags. Do you think it's wise, or should I test this workflow before? (and if yes, how? I cannot start it manually for now, although I could force it in a commit...)

Reply:
Also created the Releng PR: https://github.com/scality/releng-it/pull/504

Reply:
I think you can force the workflow via the `gh` cli if it's not available via the ui.

But overall, it's up to you, I'm confident you will make sure it's working so, feel free to merge and check afterwards.

Reply:
and my bad about the comment I think I missed a part of your file when I reviewed it 😂